### ISSUE TITLE
ARM / ARM64 Support

### ISSUE THREAD

**Issue by SaswatPadhi** [Ubiquiti-App/UNMS#185]
Hello,

Currently only `amd64` docker images are provided. Are there any plans to support `arm64` as well?

I'd love to leave UNMS running on a tiny SBC within my network. There is an independent docker repo here: https://github.com/Nico640/docker-unms, but an official build would be great!

Thanks.

**Comment by fastlorenzo** [comment]
+1 on this, it took me some time but I managed to build the unms docker for arm (not the one from the link you've sent, as it contains all services and not just unms).

The plan is to deploy on k8s, which is better if I have multiple containers.

**Comment by SaswatPadhi** [comment]
I also ended up using debian-slim instead of alpine because of musl.

Would you be willing to post your dockerfiles, @fastlorenzo ? I do have mine and would push them soon, but they're a mess right now ...

**Comment by fastlorenzo** [comment]
@SaswatPadhi I've pushed it to my repo: https://github.com/fastlorenzo/unms-chart/tree/master/docker

I'm building a Helm chart for K8S with separate containers for each service (main usage will be on RPi4)

**Comment by SaswatPadhi** [comment]
Awesome! Thanks for sharing it on GitHub.

I would let you know if I face any issues.

**Comment by fastlorenzo** [comment]
FYI it's still not fully functional as v 1.3.0-dev has a dependency on siridb, which doesn't have arm containers. Working ont it, should be fixed soon

**Comment by SaswatPadhi** [comment]
Thanks for sharing your Dockerfiles, @fastlorenzo. Your UCRM file was quite useful!

I just made mine public at https://github.com/padhi-homelab/docker_unms.

**Comment by fastlorenzo** [comment]
@SaswatPadhi awesome, is yours fully working? I still had an issue when starting the env., It was blocking saying the ucrm and unms versions were not compatible.
Might be because I was using the dev version as well 😁

**Comment by SaswatPadhi** [comment]
Yes I think it's fully working as far as I can tell. I haven't ran it for a long time though. Still testing and making sure everything works. I am running this on an `arm/v7` board. And here is the `docker-compose.yml` file I use:

https://github.com/padhi-homelab/services/tree/master/unms
